flow/cmd/handler.go

```diff
@@ -470,6 +476,9 @@ func (h *FlowRequestHandler) getWorkflowID(ctx context.Context, flowJobName stri
 	q := "SELECT workflow_id FROM flows WHERE name = $1"
 	var workflowID string
 	if err := h.pool.QueryRow(ctx, q, flowJobName).Scan(&workflowID); err != nil {
+		if errors.Is(err, pgx.ErrNoRows) {
+			return "", exceptions.NewNotFoundApiError(fmt.Sprintf("flow job %s not found", flowJobName))
```

Comment:
but then this one breaks the pattern with public/private. Maybe something like ErrNotFound and construct the message in the caller?